Raven.Database/Server/Controllers/StreamsController.cs

```diff
@@ -246,69 +246,67 @@ public async Task<HttpResponseMessage> Exploration(string collection)
 
             var msg = GetEmptyMessage();
 
-            using (var cts = new CancellationTokenSource())
+            var cts = new CancellationTokenSource();
```

Comment:
Shouldn't we dispose of the Cts? maybe we need to override the Dispose of the HttpContext and dispose of the Cts there.

Reply:
No, `StreamQueryContent` will handle that for us.